tests/qhub-config-yaml-files-for-upgrade/qhub-config-do-310-customauth.yaml

```diff
@@ -22,23 +22,23 @@ security:
     admin:
       gid: 101
 default_images:
-  jupyterhub: quansight/qhub-jupyterhub:v0.3.10
-  jupyterlab: quansight/qhub-jupyterlab:v0.3.10
-  dask_worker: quansight/qhub-dask-worker:v0.3.10
-  dask_gateway: quansight/qhub-dask-gateway:v0.3.10
+  jupyterhub: quansight/nebari-jupyterhub:v0.3.10
+  jupyterlab: quansight/nebari-jupyterlab:v0.3.10
+  dask_worker: quansight/nebari-dask-worker:v0.3.10
+  dask_gateway: quansight/nebari-dask-gateway:v0.3.10
```

Comment:
This will need to point to the nebari-dev container registry + update no? these are massively outdated

Reply:
See issue #1512